packages/nc-gui-v2/pages/signup/[[token]].vue

```diff
@@ -43,17 +44,23 @@ const formRules = {
     {
       validator: (_: unknown, v: string) => {
         return new Promise((resolve, reject) => {
-          if (isEmail(v)) return resolve(true)
+          if (!v?.length || isEmail(v)) return resolve(true)
```

Comment:
Wouldn't this accept *any* type of string now, regardless of being a valid email or not?
The conidition is now `if (string has length OR string is email-like) then { ... }`

Reply:
@bcakmakoglu when string has no length this validator will be disabled effectively which is required since there is a validator above this which checks if a string is empty. This is a fix for the case of empty string where both of the validation error message comes up.
<img width="683" alt="image" src="https://user-images.githubusercontent.com/35809690/185324569-af1108d2-0346-44bb-baa6-eddaafebb189.png">